runtime/onert/backend/cpu/CMakeLists.txt

```
@@ -16,6 +17,7 @@ target_link_libraries(${LIB_ONERT_BACKEND_CPU} PRIVATE ndarray)
 
 set_target_properties(${LIB_ONERT_BACKEND_CPU} PROPERTIES OUTPUT_NAME backend_cpu)
 
+set_target_properties(${LIB_ONERT_BACKEND_CPU} PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

```suggestion
set_target_properties(${LIB_ONERT_BACKEND_CPU} PROPERTIES POSITION_INDEPENDENT_CODE ON)

```

It would be better to read to put this line to the former line 18 (set_target_...), rather than line 21 (if...)